### PR TITLE
Add release notes for 0.243.3

### PR DESCRIPTION
**Pull request by prithvip**
Test plan - Built the docs and made sure they looked ok

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by caithagoras — APPROVED** [review]

File: presto-docs/src/main/sphinx/release/release-0.243.3.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+===============
+Release 0.243.3
+===============
+
+General Changes
+---------------
+* Fix access control checks for columns in USING clause (:pr:`15333`).
```
Comment (caithagoras) [inline presto-docs/src/main/sphinx/release/release-0.243.3.rst:7]: Use double back quotes:
```
``USING``
```

File: presto-docs/src/main/sphinx/release/release-0.243.rst [hunk 2]
```diff
@@ -5,6 +5,9 @@ Release 0.243
 .. warning::
     There is a bug in LambdaDefinitionExpression canonicalization introduced since 0.238. For more details, go to :issue:`15424`.
 
+.. warning::
+    There is a bug that results in columns in the USING clause of a query to not be checked properly for column-level access control (:pr:`15333`).
```
Comment (caithagoras) [inline presto-docs/src/main/sphinx/release/release-0.243.rst:9]: Use double back quotes:
```
``USING``
```

File: presto-docs/src/main/sphinx/release/release-0.243.3.rst (hunk 1 above)
Comment (prithvip) [reply presto-docs/src/main/sphinx/release/release-0.243.3.rst]: Done

File: presto-docs/src/main/sphinx/release/release-0.243.rst (hunk 2 above)
Comment (prithvip) [reply presto-docs/src/main/sphinx/release/release-0.243.rst]: Done

**Review by caithagoras — APPROVED** [review]

**Pull request merged**